src/nvim/event/process.c

```diff
@@ -25,11 +25,6 @@
 // For PTY processes SIGTERM is sent first (in case SIGHUP was not enough).
 #define KILL_TIMEOUT_MS 2000
 
-/// Externally defined with gcov.
-#ifdef USE_GCOV
-void __gcov_flush(void);
-#endif
```

Comment:
Is this no longer needed for gcov? I think we should not rule out adding gcov back to CI.